cmd/agent/api/agent/agent_jmx.go

```diff
@@ -57,22 +50,22 @@ func getJMXConfigs(w http.ResponseWriter, r *http.Request) {
 	for name, config := range configItems {
 		m, ok := config.(map[string]interface{})
 		if !ok {
-			err = fmt.Errorf("wrong type in cache")
+			err := fmt.Errorf("wrong type in cache")
```

Comment:
Not a Go expert, but how did this code used to work before?

Reply:
different scope

Reply:
before this diff `err` was declared above and reused in the following code blocks